lta/rest_server.py

```diff
@@ -218,6 +228,99 @@ class FilesHandler(BaseLTAHandler):
         self.set_status(200)
         self.write(ret)
 
+class FilesActionsPopHandler(BaseLTAHandler):
+    """
+    FilesActionsPopHandler handles /Files/actions/pop.
+
+    Picker provides:
+        file_obj = {
+            "source": "SITE:PATH",
+            "dest": "SITE:PATH",
+            "request": "UUID",
+            "catalog": file_catalog_record
+        }
+
+    Catalog Schema:
+        catalog_file = {
+            "uuid": uuid,
+            "logical_name": "/data/exp/IceCube...",
+            "locations": [
+                {"site": "WIPAC", "path": "/data/exp/IceCube..."},
+                {"site": "NERSC", "path": "/data/archive/ad43d3...23.zip", archive: True},
+            ]
+            "file_size": 123456789,
+            "data_type": "real",
+        }
+
+    LTA DB annotates:
+        xfer_file["uuid"] = uuid1().hex
+        xfer_file["create_timestamp"] = now()
+        xfer_file["status"] = "waiting"
+    """
+
+    @lta_auth(roles=['system'])
+    async def post(self) -> None:
+        """Handle POST /Files/actions/pop."""
+        # if there aren't enough files to fill the bundle, should we force it?
+        force = self.get_argument('force', False)
+        # figure out which site is bundling files
+        src = self.get_argument('source')
+        if src not in self.sites:
+            raise tornado.web.HTTPError(400, reason="invalid source site")
+        # figure out where the bundle is headed
+        dest = self.get_argument('dest')
+        if dest not in self.sites:
+            raise tornado.web.HTTPError(400, reason="invalid dest site")
+        # figure out how big the bundle should be
+        bundle_size = self.sites[dest]["bundle_size"]
+        # get the self-identification of the claiming bundler
+        pop_body = json_decode(self.request.body)
+        # let's build up a list of things to bundle
+        src_tuples = []
+        for uuid in self.db["Files"]:
+            db_file = self.db["Files"][uuid]
+            if not (site(db_file["source"]) == src):
+                continue
+            if not (site(db_file["dest"]) == dest):
+                continue
+            if not (site(db_file["status"]) == "waiting"):
+                continue
+            src_tuples.append((uuid, db_file["catalog"]["file_size"]))
+        # we can't bundle nothing
+        if len(src_tuples) < 1:
+            self.set_status(200)
```

Comment:
`set_status(200)` is not necessary, as that is the default status.  only necessary when you want a different number.

Reply:
I added a commit to remove the unnecessary calls.